### PR TITLE
Migrate localization tooling to Lunaria

### PR DESCRIPTION
**Pull request by Yan-Thomas**
#### Description (required)

This PR moves from using our old i18n Tracker to use Lunaria instead.

A few notes:
- I'll add "Translations that need reviews" as a follow-up.
- I am seeing some subtle color differences from the variables I copied from the current i18n Tracker, but I am not sure why.

Preview it here:
https://astro-docs-seven.vercel.app/

### CODE REVIEW

**Review by delucis — COMMENTED** [review]
Looks fantastic @Yan-Thomas! Amazing to see this coming together 💜 (and to see a little bit of docs purple living on in this page 😄)

Couple of small differences I noticed between the preview and the current dashboard:

- The dictionary files (`docsearch.ts`, `nav.ts`, and `ui.ts`) are currently listed first, but in the preview are listed last. Do we have control over that? My guess is that showing them first gives those special files a bit more visibility.

- The table legend is lacking some spacing between the items:
   | Before | After |
   |---|---|
   | three emojis, each with label text spaced out with a gap | the same emojis as before, but now missing the gap |

**Comment by Yan-Thomas** [comment]
> The dictionary files (docsearch.ts, nav.ts, and ui.ts) are currently listed first, but in the preview are listed last. Do we have control over that? I guess that showing them first gives those special files a bit more visibility.

Currently, there's no feature to sort the index. Internally we first index the generic content and *then* the dictionaries, that's why it's shown at the end.

> The table legend is lacking some spacing between the items:

Yeah, that's a very recent issue since I've moved from my own rendering system to using `@lit-labs/ssr`, it seems it doesn't accept HTML entities, which was what we were using for that spacing. This legend is generated from a string to allow i18n (`'{missing_emoji} {missing_word} {outdated_emoji} {outdated_word} {done_emoji} {done_word}'`), so there are a few limitations in how we can fix this and I still haven't found a solution.

**Comment by delucis** [comment]
> Currently, there's no feature to sort the index. Internally we first index the generic content and then the dictionaries, that's why it's shown at the end.

Interesting. Any particular reason it does that instead of respecting the order in which the files are defined in the config file?

> This legend is generated from a string to allow i18n (`'{missing_emoji} {missing_word} {outdated_emoji} {outdated_word} {done_emoji} {done_word}'`), so there are a few limitations in how we can fix this

Can’t you wrap each of the emoji–word pairs in a `<span>` and `display: flex; gap: 1rem` the parent element?

**Review by delucis — APPROVED** [review]
LGTM @Yan-Thomas! Excited to move onto our next steps 🥳

File: lunaria.config.json [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+      "lang": "de",
+      "content": {
+        "location": "src/(i18n/de/nav|content/docs/de/**/*).(ts|mdx)"
+      },
+      "dictionaries": {
+        "location": "src/i18n/de/(docsearch|ui).ts"
+      }
+    },
+    {
+      "label": "Espanõl",
```
[the remark this answers is not in the document]
Comment (delucis) [reply lunaria.config.json:43]: PRs welcome! 😆

**Pull request merged**